annotation-server/test/app.e2e-spec.ts

```diff
@@ -77,17 +86,23 @@ describe('App (e2e)', () => {
         it('should return 1 medications matching a search query with guidelines', async () => {
             const getResponse = await request(app.getHttpServer())
                 .get('/medications')
-                .query({ search: 'cod', withGuidelines: 'true' });
+                .query({
+                    search: 'cod',
+                    withGuidelines: 'true',
+                    getGuidelines: 'true',
+                });
```

Comment:
Nothing you started, but I would just pass the actual booleans instead of the corresponding strings (to avoid typos etc.).

Reply:
Nice! Didn't know that was possible.